Sources/SwiftCompilerPluginMessageHandling/PluginMessages.swift

```diff
@@ -155,6 +155,7 @@ public enum PluginMessage {
       case error
       case warning
       case note
+      case remark
```

Comment:
Could you add the addition of `PluginMessage.Diagnostic.Severity` to the [release notes](https://github.com/apple/swift-syntax/blob/main/Release%20Notes/510.md)?

Reply:
Sure, done